conventions/services/search.py

```diff
@@ -50,6 +50,8 @@ def get_queryset(self) -> QuerySet:
 
         if order_by := self._get_order_by():
             queryset = queryset.order_by(*order_by)
+        else:
+            queryset = queryset.order_by("-cree_le")
```

Comment:
We need an ordered queryset any time.
Actually, it is always specified explicitely, except for `Programme`, but we don't really care about the order for this case, as we return only a few items.

Reply:
👍